### DISCUSSION TITLE
Community standards

### DISCUSSION THREAD

**Discussion by jovan-stojanovic** [skrub-data/skrub#284] [Ideas]
Maybe this is not urgent, but we may try to comply with the [community standards](https://github.com/dirty-cat/dirty_cat/community) by:

- writing a code of conduct (or reusing [sklearn's code of conduct](https://github.com/scikit-learn/scikit-learn/blob/main/CODE_OF_CONDUCT.md)?)
- writing a pull request template (useful for PR's, again good [example here](https://github.com/scikit-learn/scikit-learn/blob/main/.github/PULL_REQUEST_TEMPLATE.md))

This could be particularly useful for any upcoming sprints, where user's would find information directly on the github page.

What do you think?

**Comment by LilianBoulard** [comment]
I like the idea, though as the project is still rather small in size, we should keep it as lightweight as possible :)

I wrote the [contributing guide](https://github.com/dirty-cat/dirty_cat/blob/master/CONTRIBUTING.md) a while ago, inspired from [Atom's](https://github.com/atom/atom/blob/master/CONTRIBUTING.md).
The code of conduct seems like the next step to me, and I do like the simplicity of sklearn's !

**Comment by GaelVaroquaux** [comment]
Those are good ideas. Reusing the material from sklearn is a good way to
start (and maybe nothing more is needed)

**Comment by jovan-stojanovic** [comment]
Added code of conduct #285
